packages/cfd/src/Containers/__tests__/cfd-change-password-confirmation.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import ChangePasswordConfirmation from '../cfd-change-password-confirmation';
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+
+    return {
+        ...original_module,
+        Icon: jest.fn(() => 'mockedIcon'),
+    };
+});
+
+jest.mock('../trading-password-manager.tsx', () => {
+    const original_module = jest.requireActual('../trading-password-manager.tsx');
+
+    return {
+        ...original_module,
+        TradingPasswordManager: jest.fn(() => 'mockedTradingPasswordManager'),
```

Comment:
Where did you use this mocked component?

Reply:
Forgot to remove it. Will amend it